payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncher.kt

```diff
@@ -296,6 +304,25 @@ class GooglePayPaymentMethodLauncher @AssistedInject internal constructor(
             get() = merchantCountryCode.equals(Locale.JAPAN.country, ignoreCase = true)
     }
 
+    @Parcelize
+    data class ShippingAddressConfig @JvmOverloads constructor(
+        /**
+         * Set to true to request a full shipping address.
+         */
+        internal val isRequired: Boolean = false,
+
+        /**
+         * ISO 3166-1 alpha-2 country code values of the countries where shipping is allowed.
+         * If this object isn't specified, all shipping address countries are allowed.
+         */
+        internal val allowedCountryCodes: Set<String> = emptySet(),
+
+        /**
+         * Set to true if a phone number is required for the provided shipping address.
+         */
+        internal val phoneNumberRequired: Boolean = false
```

Comment:
Nit: We’ve been pretty inconsistent with this, but I like when booleans begin with `is` or `has`. Should we use `isPhoneNumberRequired` here?

```suggestion
        internal val isPhoneNumberRequired: Boolean = false
```